dev-db/postgresql/files/postgresql.init-9.3-r1

```diff
@@ -21,13 +26,16 @@ depend() {
     use net
     provide postgresql
 
-    if [ "$(get_config log_destination)" = "syslog" ]; then
+    if [ "$(get_config log_destination)" = "syslog" ] ; then
         use logger
     fi
 }
 
-configured_port=$(get_config port)
-: ${configured_port:=${PGPORT}}
+configured_port=${PGPORT}
+config_port() {
+    local port=$(get_config port)
+    [ -z ${port} ] || configured_port=${port}
```

Comment:
A positive statement would be easier to read here, i.e. use `-n` and `&&`.

Reply:
One of my habits is to try to ensure that each line ends with a return code of **0** under normal flow, so here I use `||` instead of `&&`. I'll do it if you insist.